redash/tasks/queries.py

```diff
@@ -71,12 +72,21 @@ def to_dict(self):
         else:
             query_result_id = None
 
+        queriesAhead = 0
+        if task_status == 'PENDING':
+            waiting = get_waiting_in_queue('queries')
+            # find our job's index, return number of queries after it
```

Comment:
Seems this comment lies :grin: (function returns number of queries _before_ requested one)